packages/gestalt-core/build.js

```diff
@@ -116,8 +116,8 @@ const cssModules = (options = {}) => {
       const opts = {
         preset: ['default', { calc: false }],
       };
-      cssnano
-        .process(Object.values(cssCache).join(''), { from: undefined }, opts)
+      cssnano(opts)
```

Comment:
Following [this upgrade guide](https://github.com/cssnano/cssnano/releases/tag/cssnano%405.0.0) and the relevant [docs page](https://cssnano.co/docs/config-file/)